src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,37 +1,120 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import React from 'react';
+import { Movie } from '../../types/Movie';
+type NewMovie = {
+  onAdd: (movie: Movie) => void;
+}
 
-export const NewMovie = () => {
+export const NewMovie: React.FC<NewMovie> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId]= useState('');
```

Comment:
Сreate only one state for movie
```suggestion
  const [movie, setMovie] = useState({
    title: '',
    .......
  });
```